COPYRIGHT

```diff
@@ -0,0 +1 @@
+Copyright 2020 University of Washington
```

Comment:
Is this COPYRIGHT correct? Shouldn't it be something like:

```
Copyright 2020 Association of Universities for Research in Astronomy, Inc. (AURA)
```

?

Reply:
This is the correct copyright for repos that I make.